rfc/2/index.md

```diff
@@ -126,15 +108,40 @@ There are a few notable changes that Zarr v3 brings for OME-Zarr:
 The Zarr specification does not prescribe the support stores for Zarr hierarchies.
 HTTP(S), File system, S3, GCS, and Zip files are commonly used stores.
 
+## Proposal
+
+This RFC proposes to adopt version 3 of the Zarr format for OME-Zarr.
+Images that use the new version of OME-Zarr metadata MUST NOT use Zarr version 2 any more.
+
 With this proposal all features of the Zarr specification are allowed in OME-Zarr.
 In the future, the OME-Zarr community MAY decide to restrict the allowed feature set.
 
+The motivation for making this hard cut is to reduce the burden of complexity for implementations.
+Currently, many Zarr library implementations support both versions.
+However, in the future they might deprecate support for version 2 or deprioritize it in terms of features and performance.
+Additionally, there are OME-Zarr implementations that have their own integrated Zarr stack.
+With this hard cut, implementations that only support OME-Zarr versions > 0.5 will not need to implement Zarr version 2 as well.
+
+From a OME-Zarr user perspective, the hard cut also makes things simpler: ≤ 0.5 => Zarr version 2 and > 0.5 => Zarr version 3.
```

Comment:
Version 0.5 is Zarr version 3, so I would use the `≥` symbol there like:
< 0.5 => Zarr version 2 and ≥ 0.5 => Zarr version 3

Reply:
Thanks!